scripts/install.tpl.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+olm_version=$OLM_V0_VERSION
+cert_mgr_version=$CERT_MGR_VERSION
+rukpak_version=$RUKPAK_VERSION
+operator_controller_version=$VERSION
+
+if [[ -z "$olm_version" || -z "$cert_mgr_version" || -z "$rukpak_version" || -z "$operator_controller_version" ]]; then
+    err="Error: Missing component version(s) for: "
+    if [[ -z "$olm_version" ]]; then
+        err+="operator-lifecycle-manager "
+    fi 
+    if [[ -z "$cert_mgr_version" ]]; then
+        err+="cert-manager "
+    fi 
+    if [[ -z "$rukpak_version" ]]; then
+        err+="rukpak "
+    fi 
+    if [[ -z "$operator_controller_version" ]]; then
+        err+="operator-controller"
+    fi 
+    echo $err
+    exit 1
+fi
+
+function kubectl_wait() {
+    namespace=$1
+    runtime=$2
+    timeout=$3
+
+    kubectl wait --for=condition=Available --namespace="${namespace}" "${runtime}" --timeout="${timeout}"
+}
+
+curl -L -s "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_version}/install.sh" | bash -s "${olm_version}"
+
+kubectl apply -f "https://github.com/cert-manager/cert-manager/releases/download/${cert_mgr_version}/cert-manager.yaml"
+kubectl_wait "cert-manager" "deployment/cert-manager-webhook" "60s"
+
+kubectl apply -f "https://github.com/operator-framework/rukpak/releases/download/${rukpak_version}/rukpak.yaml"
+kubectl_wait "rukpak-system" "deployment/core" "60s"
+kubectl_wait "rukpak-system" "deployment/helm-provisioner" "60s"
+kubectl_wait "rukpak-system" "deployment/rukpak-webhooks" "60s"
+
+kubectl apply -f "https://github.com/operator-framework/operator-controller/releases/download/${operator_controller_version}/operator-controller.yaml"
```

Comment:
One thing that could be useful here is if we used this install script as part of the `e2e` and `run` targets. That way we actually exercise this as part of our dev cycle and testing.

I think all we would need to do is make the entire operator-controller manifest path configurable.

For `make run`, we'd generate the manifest without the image ref substitution, and then this would be `kubectl apply -f ./operator-controller.yaml`

For the release manifest, we'd substitute the entire release URL in like you're doing here.

If this is not a trivial change, its totally non-blocking and would be fine to do in a follow-up.

Reply:
Cool! I'll give this a shot, it's a good idea.

Reply:
Added :)